rotkehlchen/api/v1/schemas.py

```diff
@@ -2333,3 +2334,22 @@ class DetectTokensSchema(
     OptionalAddressesListSchema,
 ):
     ...
+
+
+class UserNotesPutSchema(Schema):
```

Comment:
If I remember correctly, we don't use `patch`, `put` and other verbs in schemas names